crawl-ref/source/mon-data.h

```diff
@@ -6960,6 +6974,19 @@ DUMMY(MONS_HELL_LORD, '&', COLOUR_UNDEF, "hell lord", TILEP_MONS_PROGRAM_BUG)
     {TILEP_MONS_AGNES}, TILE_ERROR
 },
 
+{
+    MONS_CIRI, 'i', BROWN, "Ciri",
+    M_UNIQUE | M_HERD | M_WARM_BLOOD | M_SPEAKS | M_FEMALE | M_FIGHTER,
```

Comment:
M_HERD doesn't work without some tweaks. If you look at the relevant code in `mon-movetarget.cc` (both calls to `herd_monster()`), you'll notice that all of the logic checks for mons_genus. If you want these to herd together strongly, you'll need to change some code. (But it's not clear to me how critical it is for them to herd together, especially if they're in chaotic situations like the Abyss and Pan where the player won't be revisiting th same level repeatedly.)